src/workerd/api/streams/internal.c++

```diff
@@ -1565,9 +1585,10 @@ jsg::Promise<void> WritableStreamInternalController::writeLoopAfterFrontOutputLo
   };
 
   const auto maybeAbort = [this](jsg::Lock& js, auto& request) -> bool {
-    auto& writable = state.get<Writable>();
+    auto& writable = state.get<IoOwn<Writable>>();
```

Comment:
Not new in this PR, but should we use `KJ_ASSERT_NONNULL(state.tryGet` here? `get` is UB if for whatever reason the state is a different type. IMO it should only ever be used if the type was checked literally right before.